.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build CI
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
+    - name: Translate Repo Name For Build Tools filename_prefix
+      id: repo-name
+      run: |
+        echo ::set-output name=repo-name::$(
+        echo ${{ github.repository }} |
+        awk -F '\/' '{ print tolower($2) }' |
+        tr '_' '-'
+        )
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Versions
+      run: |
+        python3 --version
+    - name: Checkout Current Repo
+      uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Checkout tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: adafruit/actions-ci-circuitpython-libs
+        path: actions-ci
+    - name: Install deps
+      run: |
+        source actions-ci/install.sh
+    - name: Library version
+      run: git describe --dirty --always --tags
+    - name: PyLint
+      run: |
+        pylint $( find . -path './adafruit_io*.py' )
```

Comment:
Now that we've decided on and merged the updates to the `build.yml` in the cookiecutter, they should be applied here rather than have this repo have a custom `build.yml` because it shouldn't need it. 

Since adabot won't be able to patch this as is, it'll need to be patched by hand anyways, so we might as well do it now. Otherwise we'd need to put in a separate PR to do what the adabot patch will, or merge the failing "old" build.yml and let adabot patch it. I prefer having a clean merge